.gitignore

```diff
@@ -3,6 +3,7 @@ bazel-*
 .idea/
 .ijwb/
 node_modules
+.github/pull_request_template.md
```

Comment:
that's an odd one?

Reply:
graphite command line tools?

Reply:
You got it. It reads from the file and puts it into your command line editor before it creates the PR